docs/sources/setup-grafana/start-restart-grafana.md

```diff
@@ -0,0 +1,193 @@
+```
+
+#### Restart the Grafana server using init.d
+
+To restart the Grafana server, run the following commands:
+
+```bash
+Need input
+```
+
+### Start the server using the binary
+
+The `grafana-server` binary .tar.gz needs the working directory to be the root install directory where the binary and the `public` folder are located.
+
+To start the Grafana server, run the following command:
+
+```bash
+./bin/grafana-server web
+```
+
+## Docker
+
+To restart the Grafana service, use the `docker restart` command.
+
+`docker restart grafana`
+
+Alternately, you can use the `docker compose restart` command to restart Grafana. For more information, refer to [docker compose documentation](https://docs.docker.com/compose/).
+
+### Docker compose example
+
+Configure your `docker-compose.yml` file. For example:
+
+```bash
+grafana:
+  image: grafana/grafana:latest
+  ports:
+    - "3000:3000"
+  environment:
+    - TERM=linux
+    - GF_INSTALL_PLUGINS=grafana-clock-panel,grafana-piechart-panel,grafana-polystat-panel
+```
+
+Start the Grafana server:
+
+`docker-compose up`
+
+This starts the Grafana server along with the three plugins specified in the YAML file.
+
+To restart the running container, use this command:
+
+`docker-compose restart grafana`
+
+## Windows
+
+Complete the following steps to start the Grafana server on Windows:
+
+1. Execute `grafana-server.exe`, which is located in the `bin` directory.
+
+   We recommend that you run `grafana-server.exe` from the command line.
+
+   If you want to run Grafana as a Windows service, you can download [NSSM](https://nssm.cc/).
+
+1. To run Grafana, open your browser and go to the Grafana port (http://localhost:3000/ is default).
+
+   > **Note:** The default Grafana port is `3000`. This port might require extra permissions on Windows. If it does not appear in the default port, you can try changing to a different port.
+
+1. If you need to change the port, complete the following steps:
+
+   a. In the `conf` directory, copy `sample.ini` to `custom.ini`.
+
+   > **Note:** You should edit `custom.ini`, never `defaults.ini`.
+
+   b. Edit `custom.ini` and uncomment the `http_port` configuration option (`;` is the comment character in ini files) and change it to something similar to `8080`, which should not require extra Windows privileges.
+
+To restart the Grafana server, complete the following steps:
+
+1. Open the **Services** app.
+1. Right-click on the **Grafana** service.
+1. In the context menu, click **Restart**.
+
+## macOS
+
+Restart methods differ depending on whether you installed Grafana using Homebrew or as standalone macOS binaries.
+
+### Restart Grafana using Homebrew
+
+Use the [Homebrew](http://brew.sh/) restart command:
+
+```bash
+brew services restart grafana
+```
+
+### Restart standalone macOS binaries
+
+To restart Grafana:
+
+1. Open a terminal and go to the directory where you copied the install setup files.
+1. Run the command:
+
+```bash
+./bin/grafana-server web
```

Comment:
Same here (and anywhere else it appears), remove `web` parameter as it doesn't do anything.